examples/gno.land/p/demo/ufmt/ufmt.gno

```diff
@@ -129,6 +135,66 @@ func Sprintf(format string, args ...interface{}) string {
 			default:
 				buf += "(unhandled)"
 			}
+		case "x":
+			switch v := arg.(type) {
+			case uint8:
+				buf += strconv.FormatUint(uint64(v), 16)
+			case []uint8, [32]uint8:
+				for _, b := range v {
+					buf += strconv.FormatUint(uint64(b), 16)
+				}
+			default:
+				buf += "(unhandled)"
+			}
```

Comment:
this code... isn't supposed to work... https://go.dev/play/p/wzaZ2BdezvX

Reply:
yeah ok, it doesn't in gno, either. that's why the build is failing.

Reply:
Hmm, it doesn't work anymore. I've just removed that lines.